dss/__init__.py

```diff
@@ -26,14 +26,14 @@
 # CONSTANTS COMMON TO THE INDEXER AND QUERY ROUTE.
 
 # ES index containing all docs
-DSS_ELASTICSEARCH_INDEX_NAME = "hca"
+DSS_ELASTICSEARCH_INDEX_NAME = "hca-" + os.environ["DSS_DEPLOYMENT_STAGE"]
```

Comment:
Because the DSS is intended to be project independent, we should use a more general index base name than "hca". Suggestions are welcome. "index"?,  "primary"? ...
Yet, that may be best done as a different PR.

Reply:
yeah I thought that too. "dss-docs" might do.